### PR TITLE
Added: Customer and executive pages

### PR DESCRIPTION
**Pull request by lokeshkose**
Implemented:  Customer and Executive list pages.

### CODE REVIEW

File: src/services/Api.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import axios, { AxiosError, AxiosInstance } from 'axios';
 import { message } from 'antd';
 import { NavigateFunction } from 'react-router-dom';
 const BASE_URL = process.env.REACT_APP_BASE_URL;
+console.log(process.env);
```
Comment (LalitKushwah) [inline src/services/Api.ts]: Please remove this log

File: src/common/Users/Users.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+export const Users: React.FC<UsersProps> = (props) => {
+  const [searchByValue, setSearchByValue] = useState<string>();
+  const [province, setProvince] = useState<string>();
+  const [isEdit, setIsEdit] = useState<boolean>(false);
+  const [user, setUser] = useState<UpdateUser>({} as UpdateUser);
+  const onSearchByChangeHandler = (e: ChangeEvent<HTMLInputElement>) => {
+    setTimeout(() => {
+      setSearchByValue(e.target.value);
```
Comment (LalitKushwah) [inline src/common/Users/Users.tsx:35]: Why we are using timeout here?

File: src/pages/SalesExecutive/SalesExecutive.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, { ChangeEvent } from 'react';
+import './SalesExecutive.css';
+import { Users } from '../../common/Users/Users';
+import { FileExcelOutlined, PlusCircleOutlined } from '@ant-design/icons';
+import { PAGE_TYPE } from '../../common/Users/utils';
+
+export const SalesExecutive: React.FC = () => {
+  return (
+    <Users
+      pageTitle='Salesmans'
```
Comment (LalitKushwah) [inline src/pages/SalesExecutive/SalesExecutive.tsx]: This texts should come from i18n

File: src/common/Users/Data.tsx [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+    {
+      key: '2',
+      label: (
+        <DeleteWithRBAC
+          arrow={false}
+          placement='left'
+          key='confirm'
+          className='Popconfirm-Button'
+          title={t('users.confirmation')}
+          description={`${t('deleteConfirmationDesc')} ${selectedUser.name}?`}
```
Comment (LalitKushwah) [inline src/common/Users/Data.tsx]: this should be user.deleteConfirmationDesc

File: src/common/Users/Data.tsx [hunk 5]
```diff
@@ -0,0 +1,412 @@
...
+          buttontext={t('users.delete')}
+          okText={t('users.yes')}
+          cancelText={t('users.no')}
+          onConfirm={() => {
```
Comment (LalitKushwah) [inline src/common/Users/Data.tsx]:
```suggestion
          onConfirm={onDeleteUser}
```

File: src/common/Users/Data.tsx [hunk 6]
```diff
@@ -0,0 +1,412 @@
...
+        <ResetPassworWithRBAC
+          placement='left'
+          key='confirm'
+          className='Popconfirm-Button'
+          title={t('users.confirmation')}
+          description={`${t('users.resetPasswordDesc')} $${selectedUser.name}?`}
+          buttontext={t('users.resetPassword')}
+          okText={t('users.yes')}
+          cancelText={t('users.no')}
+          onConfirm={() => {
```
Comment (LalitKushwah) [inline src/common/Users/Data.tsx]:
```suggestion
          onConfirm={onResetPassword}
```

File: src/common/Users/Users.tsx (hunk 2 above)
Comment (lokeshkose) [reply src/common/Users/Users.tsx:35]: it is for debounce.

File: src/services/Api.ts (hunk 1 above)
Comment (lokeshkose) [reply src/services/Api.ts]: Removed.

File: src/common/Users/Users.tsx (hunk 2 above)
Comment (lokeshkose) [reply src/common/Users/Users.tsx:35]: added clearTimeout()

File: src/pages/SalesExecutive/SalesExecutive.tsx (hunk 3 above)
Comment (lokeshkose) [reply src/pages/SalesExecutive/SalesExecutive.tsx]: Done

File: src/common/Users/Data.tsx (hunk 4 above)
Comment (lokeshkose) [reply src/common/Users/Data.tsx]: Done

File: src/common/Users/Data.tsx (hunk 5 above)
Comment (lokeshkose) [reply src/common/Users/Data.tsx]: done

File: src/common/Users/Data.tsx (hunk 6 above)
Comment (lokeshkose) [reply src/common/Users/Data.tsx]: done

**Review by LalitKushwah — APPROVED** [review]

**Pull request merged**